worldedit-core/src/main/java/com/sk89q/worldedit/world/storage/ZippedMcRegionChunkStore.java

```diff
@@ -84,9 +84,14 @@ protected InputStream getInputStream(String name, String worldName) throws IOExc
             for (Enumeration<? extends ZipEntry> e = zip.entries(); e.hasMoreElements(); ) {
                 ZipEntry testEntry = e.nextElement();
                 // Check for world
-                if (testEntry.getName().startsWith(worldName + "/")) {
-                    if (pattern.matcher(testEntry.getName()).matches()) { // does entry end in .mca
-                        folder = testEntry.getName().substring(0, testEntry.getName().lastIndexOf('/'));
+                String entryName = testEntry.getName();
+                if (entryName.startsWith(worldName + "/")) {
```

Comment:
on second look this line is still broken, it doesn't use the matcher like the TrueZip version does.